adapter/config/default_config.go

```diff
@@ -146,6 +146,30 @@ var defaultConfig = &Config{
 			AllowPartialMessage: false,
 			PackAsBytes:         false,
 		},
+		Filters: filters{
+			Compression: compression{
+				Enabled: false,
```

Comment:
We can enable compression bydefault for the main branch I guess (Since it is going to be a new version anyway).
@pubudu538 @AmaliMatharaarachchi wdyt?

Reply:
If we are changing, lets update the config.toml.template too.

Reply:
+1